src/index/base.cpp

```diff
@@ -262,13 +264,27 @@ void BaseIndex::BlockConnected(const std::shared_ptr<const CBlock>& block, const
                       best_block_index->GetBlockHash().ToString());
             return;
         }
-        if (best_block_index != pindex->pprev && !Rewind(best_block_index, pindex->pprev)) {
+
+        // If best_block_index is an ancestor of pindex, then this is a block that we've already
```

Comment:
In commit "index: ignore BlockConnected if pindex is in chain" (270579f85fbe8cf97a9f506a258ad3ed865fcbeb)

Comment is a little off. Should say "If pindex in an ancestor of best_block_index" not "If best_block_index is an ancestor of pindex"